CMakeLists.txt

```diff
@@ -172,6 +172,12 @@ add_cxx_flag(-O0 DEBUG)
 add_cxx_flag(-ggdb DEBUG)
 add_cxx_flag(-DDEBUG DEBUG)
 
+add_c_flag(-ggdb RELWITHDEBINFO)
```

Comment:
Wait what???
I see this just now.
This is the job of cmake, so is the above case for DEBUG